modules/gdscript/tests/scripts/parser/errors/class_name_after_annotation.out

```diff
@@ -1,2 +1,2 @@
 GDTEST_PARSER_ERROR
-Annotation "@icon" is not allowed in this level.
```

Comment:
This is just a test. It does not affect the message (but must match it). You need this code:

https://github.com/godotengine/godot/blob/3275428b1f8294242b5358fdb47822a89a425d8e/modules/gdscript/gdscript_parser.cpp#L1437

Reply:
Ah, ok. I just tried to search for the current error and replaced it with a new one. I'll try to find some time later today to actually build and test the changes myself, I shouldn't leave it to others. I just have never successfully built anything c++, so wish me luck! 😅